amptorch/trainer.py

```diff
@@ -221,6 +229,7 @@ def train(self, raw_data=None):
         self.net.fit(self.train_dataset, None)
         elapsed_time = time.time() - stime
         print(f"Training completed in {elapsed_time}s")
+        return elapsed_time
```

Comment:
We can probably remove this, I imagine you had it for testing purposes.

Reply:
done